client/src/App.tsx

```diff
@@ -21,8 +22,9 @@ function App(): JSX.Element {
             <SocketProvider>
               <NavBar />
               <Switch>
-                <Route exact path="/login" component={Login} />
+                <Route exact path="/landingpage" component={Landingpage} />
```

Comment:
```suggestion
                <Route exact path="/landing-page" component={Landingpage} />
```
Please use hyphenated case for route names